explore/src/main/scala/explore/users/UserPreferencesPopup.scala

```diff
@@ -194,6 +194,24 @@ object UserPreferencesContent:
       ) =>
         import ctx.given
 
+        val layoutsPopup = ctx.toastRef.get.flatMap(
```

Comment:
@rpiaggio I tried `useToastRef` and it all compiles but no toast is shown 🤷

Reply:
I'll take a look

Reply:
After taking a look, I remembered that `useToastRef` is only used to create a ref to use wherever the `Toast` is defined.

After that, a wrapper `ToastCtx` is stored in the `AppContext`, which can be obtained with the `ToastCtx[F]` summoner when an implicit `AppContext` is in scope.

I changed the code to make use of this, as well as adding a method to the `ToastCtx` API that receives a `MessageItem`.

I inadvertently pushed to this branch instead of creating a new one, sorry for that.

Reply:
Cool thanks